backends-velox/src/test/scala/org/apache/gluten/execution/MiscOperatorSuite.scala

```diff
@@ -239,6 +239,13 @@ class MiscOperatorSuite extends VeloxWholeStageTransformerSuite with AdaptiveSpa
       "select l_orderkey from lineitem " +
         "where l_partkey in (1552, 674) or l_partkey in (1552) and l_orderkey > 1") { _ => }
     checkLengthAndPlan(df, 73)
+
+    runQueryAndCompare(
```

Comment:
We have merged https://github.com/apache/incubator-gluten/commit/d6326f06c8158de2eb08bb9ba7e3e4eb6e3964ef. Would you like to try if this issue has been fixed? Perhaps we can merge this test to ensure the functionality. Thanks for your patience.

Reply:
Thank you, I will try it locally later.